src/main/java/seedu/bankwithus/user/AccountList.java

```diff
@@ -124,13 +126,19 @@ public String askUserForBalance() {
             if (balance < 0) {
                 throw new NegativeAmountException();
             }
+            if (getMainAccount().isMoreThanTwoDecimal(balance)){
```

Comment:
I would presume this Line is what causes the program to crash

because when the program runs for the first time, and there is no main account and the size of the [private ArrayList<Account> accounts] is 0 the getMainAccount will throw an index out of bounds exception when u try to access index 1 of an arrayList of size 0. I think once u solve this should be fine